contrib/tvmop/core/umath.py

```diff
@@ -0,0 +1,113 @@
+ # Licensed to the Apache Software Foundation (ASF) under one
+ # or more contributor license agreements.  See the NOTICE file
+ # distributed with this work for additional information
+ # regarding copyright ownership.  The ASF licenses this file
+ # to you under the Apache License, Version 2.0 (the
+ # "License"); you may not use this file except in compliance
+ # with the License.  You may obtain a copy of the License at
+ #
+ #   http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing,
+ # software distributed under the License is distributed on an
+ # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ # KIND, either express or implied.  See the License for the
+ # specific language governing permissions and limitations
+ # under the License.
+import tvm
+from .. import defop, AllTypes
+
+def compute_floor_divide(dtype, ndim):
+    A = tvm.placeholder([tvm.var() for _ in range(ndim)], name='A', dtype=dtype)
+    B = tvm.placeholder([tvm.var() for _ in range(ndim)], name='B', dtype=dtype)
+    if dtype in ['float16', 'float32', 'float64']:
+        C = tvm.compute([tvm.var() for _ in range(ndim)],
+                    lambda *index: tvm.floor(A[index] / B[index]), name='C')
+    else:
+        C = tvm.compute([tvm.var() for _ in range(ndim)],
+                        lambda *index: tvm.if_then_else(B[index] == 0, tvm.const(0, dtype),
+                                                        tvm.floor(A[index].astype('float64') /
+                                                                  B[index].astype('float64')).astype(dtype)), name='C')
+    s = tvm.create_schedule(C.op)
+    return s, A, B, C
+
+@defop(name="floor_divide", target="cpu", auto_broadcast=True,
+       dtype=AllTypes, ndim=list(range(6)))
+def floor_divide(dtype, ndim):
```

Comment:
We can extract a `default_schedule` since all of the `cpu` schedule look the same. Also for GPU schedules.